types/mongodb/index.d.ts

```diff
@@ -928,8 +928,9 @@ export interface Collection<TSchema = Default> {
     deleteOne(filter: FilterQuery<TSchema>, options?: CommonOptions & { bypassDocumentValidation?: boolean }): Promise<DeleteWriteOpResultObject>;
     deleteOne(filter: FilterQuery<TSchema>, options: CommonOptions & { bypassDocumentValidation?: boolean }, callback: MongoCallback<DeleteWriteOpResultObject>): void;
     /** http://mongodb.github.io/node-mongodb-native/3.1/api/Collection.html#distinct */
+    distinct(key: string, callback: MongoCallback<any>): void;
```

Comment:
Would you please specify the type of the callback at least? or use generic from distinct or the Collection?

Reply:
The callback can return `any` as you can specify any key and it can any data-type in the database.

We could potentially make it check if it can extract `TSchema[key]`, but as this PR doesn't touch the callback (this line is just duplicated from the ones below) I think that that belongs in a separate PR.

Reply:
Actually, more correct is probably `any[]`. But I'd rather submit that as a second PR as it isn't related to the changes I did now...